docs/how_to/introduction_to_deep_learning_guide.md

```diff
@@ -0,0 +1,49 @@
+# Introduction to Deep Learning Guide
```

Comment:
This type of introductory content on deep learning is not the suitable for documentation. The assumption is the end user knows what ML is. If not, there are other resources that can help them understand ML.